src/ui/wxWidgets/DbSelectionPanel.cpp

```diff
@@ -124,7 +126,10 @@ DbSelectionPanel::DbSelectionPanel(wxWindow* parent,
 
 DbSelectionPanel::~DbSelectionPanel()
 {
-  delete m_pollingTimer;
+  if (m_pollingTimer != nullptr) {
```

Comment:
The if statement isn't needed, since the language guarantees that delete-ing a nullptr is safe.
Also, I don't think that setting m_pollingTimer to nullptr is needed, since when the destructor function is done, nothing should be able to access this member.
(This comment applies to all destructors)